articles/aggregation-in-dbms/index.md

```diff
@@ -0,0 +1,78 @@
+
+This article takes you through important aspects regarding aggregation in DBMS. It also explains why it is needed and how it takes place. 
+
+### Introduction to Database Management Systems (DBMS)
+Before learning what DBMS is, we need to understand two important terms: data and database.Data refers to facts, figures or statistics gathered for various uses, including analysis or future reference. A database refers to a collection of organized or related data. A good example of a database is a collection of cities within a country. This is regarded as a database because all the data in that collection is related. 
+
+A Database Management System (DBMS) is a software package that is used for managing databases. A DBMS facilitates operations such as accessing, updating, administering, fetching, manipulating, creating and deleting the database system. It also enables users to share data with other applications or users. The main aim of this software is to enhance the storage and retrieval of data. Some of the examples of database management systems include MySQL, Oracle, SQLite, Ms-Access, and MariaDB. 
+
+The following are the main characteristics of DBMS
+* **Access:** DBMS enables multiple users to gain access to the same data. 
+* **Sharing:** It enables users to share data with other applications and users. 
+* **Security:** Data sharing is done in the form of data encryptions, which ensure that there is no security threat. 
+* **Back up:** DBMS has a backup and recovery feature that enables it to restore the consistent state in the event of a system crash. 
+* **Data independence:** Any changes to data do not affect the application.
+* **Restricted Access:** A DBMS consists of defined roles, which enhances restricted access for specific data. This boosts the level of data security within the system. 
+
+### Aggregation in DBMS
+Aggregation refers to the process by which entities are combined to form a single meaningful entity. The specific entities are combined because they do not make sense on their own. To establish a single entity, aggregation creates a relationship that combines these entities. The resulting entity makes sense because it enables the system to function well. 
+
+When using data in the form of numerical values, the following operations can be used to perform DBMS aggregation;
+* **Average (AVG):** This function provides the mean or average of the data values. 
+* **Sum:** This provides a total value after the data values have been added. 
+* **Count:** This provides the number of records.
+* **Maximum (Max):** This function provides the maximum value of a given set of data.
+* **Minimum (Min):** This provides the minimum value of a given set of data. 
+* **Standard deviation (std dev):** This provides the dispersion or variation of the sets of data.

```

Comment:
> **Standard deviation (std dev):** This provides the dispersion or variation of the sets of data.

You could add a line on how we can infer things from the standard deviation of a database. For example, 
Database of student marks.

If the standard deviation is high, it means the average is obtained by lower number of students than usual, and the lowest and highest marks is higher.

Reply:
Done.